docker-compose-mongo.yml

```diff
@@ -40,6 +41,7 @@ services:
         dos2unix /etc/cron.d/manage-volume-cron
         chmod 644 /etc/cron.d/manage-volume-cron
         systemctl restart cron
+        echo "$MONGO_DB_KEYFILE_STRING" > /data/keyfile-import.txt
```

Comment:
Should we echo directly to /data/keyfile.txt and get rid of the cp line directly below?